podpac/core/node.py

```diff
@@ -426,9 +457,9 @@ def get_cache(self, key, coordinates=None):
         if not self.has_cache(key, coordinates=coordinates):
             raise NodeException("cached data not found for key '%s' and cooordinates %s" % (key, coordinates))
 
-        # return cache.get(self, data, key, coordinates=coordinates)
+        return self.cache_ctrl.get(self, key, coordinates=coordinates)
 
-    def put_cache(self, data, key, coordinates=None, overwrite=False):
+    def put_cache(self, data, key, coordinates=None, overwrite=False, raise_no_cache_exception=True):
```

Comment:
I really went back and forth on this one. I need the `raise_no_cache_exception` for when I REALLY want to make sure I cache something (Then I can set it to true). but I don't want my nodes to fail whenever I don't have a cache defined. The decorator will still try to put... perhaps I should handle this elsewhere? But I think it suffices for now.